pkg/model/master_volumes.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	DefaultEtcdVolumeSize    = 20
+	DefaultEtcdVolumeSize    = 40
```

Comment:
So the value thats open, should be tied to the value above in  pkg/model/awsmodel/autoscalinggroup.go. Although like you had said disk is cheap and i figured i'd increase this as well.